backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc

```diff
@@ -25,6 +25,11 @@ include::modules/migration-configuring-aws-s3.adoc[leveloffset=+1]
 include::modules/oadp-about-backup-snapshot-locations-secrets.adoc[leveloffset=+1]
 include::modules/oadp-creating-default-secret.adoc[leveloffset=+2]
 include::modules/oadp-secrets-for-different-credentials.adoc[leveloffset=+2]
+include::modules/oadp-ssec-encrypted-backups.adoc[leveloffset=+2]
+
+You can also download the file with the additional encryption key backed up with Velcro by running a different command. See xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc#oadp-ssec-encrypted-backups-velero_installing-oadp-aws[Downloading a file with an SSE-C encryption key for files backed up by Velero].
```

Comment:
This could be an additional resource. Otherwise, you have in running text a reference to context from the previous module that can't stand on its own.

Reply:
@maxwelldb I'm trying to work around the rule against xrefs in modules. I want the user reading oadp-ssec-encrypted-backups.adoc to be able to click a link to oadp-ssec-encrypted-backups-velero.adoc. Since I can't put an xref in oadp-ssec-encrypted-backups.adoc, I decided to put an xref in the assembly that includes oadp-ssec-encrypted-backups.adoc.  The reader will not notice that they are now reading the assembly. If there is a better way to make the xref, please let me know. One way or the other, we do want to point the reader to the Velero information. Thanks.

Reply:
@maxwelldb Hadn't thought of that -- thanks. Although "Additional resources" sections don't usually include text, I found a fairly important one that does: https://redhat-documentation.github.io/modular-docs/#what-modular-documentation-is .

I do think in this case that introductory text is needed, but I'm open to suggestions about the wording -- or if you disagree. My main concern is pointing users to the Velero information in a way that fits RH and OpenShift style.